en/docs/chapter_graph/graph.md

```diff
@@ -78,6 +78,6 @@ As shown in the table below, many real-world systems can be modeled with graphs,
 
 |                 | Vertices         | Edges                                         | Graph Computing Problem          |
 | --------------- | ---------------- | --------------------------------------------- | -------------------------------- |
-| Social Networks | Users            | Friendships                                   | Potential Friend Recommendations |
+| Social Networks | Users            | Follow / Followed                                   | Potential Following Recommendations |
```

Comment:
Probably a better word is "Connections", which covers bidirectional friendships in Facebook and directed "following and follower" in Twitter/Instagram.

Reply:
I still think follow and followed is better as it is more coherent with previous statements regarding this topic

Reply:
It is also consistent with the terms used in these specific software